jekyll/faq.md

```diff
@@ -68,6 +61,10 @@ glue code thanks to powerful metaprogramming capabilities of Nim.
 - Micro: Included
 - Atom: [https://atom.io/packages/nim](https://atom.io/packages/nim)
 
+## What have been the major influences in the language's design?
+
+The language borrows heavily from (in order of impact): Modula 3, Delphi,
+Ada, C++, Python, Lisp, Oberon.
```

Comment:
i think it'd be fair to add D somewhere in that list (I don't care where in the list)
eg of similar features:
UFCS ~ method call syntax
CTFE
powerful template metaprogramming
mixin
user defined attributes ~ pragma annotations